vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageVoiceItem.kt

```diff
@@ -55,10 +55,12 @@ abstract class MessageVoiceItem : AbsMessageItem<MessageVoiceItem.Holder>() {
     var waveform: List<Int> = emptyList()
 
     @EpoxyAttribute
-    var izLocalFile = false
+    @JvmField
+    var isLocalFile = false
```

Comment:
Should we really keep these fields which you annotated with `@JvmFields`. We could use `private const` instead or directly the boolean value where it is called. Same remark for the field `isLocalFile` in `MessageAudioItem`. What do you think?

Reply:
These can't be `const` as they're set via epoxy, but the reason for adding `@JvmField` here is Epoxy returns a compile error without it.

What I find weird is that this only happened after renaming it to fix the spelling. The reasoning for that is something that completely baffles me 🤷

Reply:
I agree. I prefer good naming as well.

Reply:
OK